proto/topo.proto

```diff
@@ -81,6 +81,32 @@ message Node {
   // If interfaces is empty the interfaces defined in the links portion of the
   // topology will be populated into the node.
   map<string, Interface> interfaces = 12;
+  // Any host constraints required by the node. These may not be required by a 
+  // vendor, but it can be used to provide requirements for the node.
+  repeated HostConstraint host_constraints = 13;
+}
+
+// HostConstraint is a constraint on the host where the node is running.
+message HostConstraint{
+  oneof constraint{
+    // Includes the required sysctl settings that need to be configured
+    KernelParam sysctl_constraint = 1;
```

Comment:
I would consider changing the name sysctl_constraint to kernel_param_constraint or kernel_constraint

otherwise this LGTM

Reply:
Addressed.